src/topologies/halfedge.jl

```diff
@@ -97,33 +97,34 @@ struct HalfEdgeTopology <: Topology
   edge4pair::Dict{Tuple{Int,Int},Int}
 end
 
-function HalfEdgeTopology(halves::AbstractVector{Tuple{HalfEdge,HalfEdge}})
-  # make sure that first half-edge is in the interior
-  ordered = [isnothing(h₁.elem) ? (h₂, h₁) : (h₁, h₂) for (h₁, h₂) in halves]
-
-  # flatten pairs of half-edges into a vector
-  halfedges = [half for pair in ordered for half in pair]
-
-  # map element and vertex to a half-edge
+function HalfEdgeTopology(halves::AbstractVector{Tuple{HalfEdge,HalfEdge}}, nelems::Int)
```

Comment:
Could we add a default value to `nelems` in case someone wants to construct the topology from the list of `halves` alone? I understand `nelems` is only used in `sizehint!`.